buildconfig/pygame-stubs/pixelarray.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Tuple, Union, List, Optional
+
+from pygame.color import Color
+from pygame.surface import Surface
+
+_ColorValue = Union[Color, Tuple[int, int, int], List[int], int, Tuple[int, int, int, int]]
+
+class PixelArray:
+    surface: Surface
+    itemsize: int
+    ndim: int
+    shape: Tuple[int, ...]
+    strides: Tuple[int, ...]
+    def __init__(self, surface: Surface) -> None: ...
+    def make_surface(self) -> Surface: ...
+    def replace(self, color: _ColorValue, repcolor: _ColorValue, distance: Optional[float]=0, weights: Optional[Tuple[float, float, float]]=(0.299, 0.587, 0.114)) -> None: ...
```

Comment:
Would be easier to read with shorter lines.

`weights` can be any sequence with 3 elements. See pixelarray_methods.c@_get_weights